src/main/java/seedu/fridgefriend/command/HistoryCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.fridgefriend.command;
+
+import seedu.fridgefriend.utilities.LoggingHandler;
+import seedu.fridgefriend.utilities.Storage;
+import seedu.fridgefriend.utilities.Ui;
+
+/**
+ * Represents a command to read the history data from the history textfile.
+ * History textfile is stored by default in data/historyData.txt.
+ * Food data is automatically added into history textfile,
+ * after every AddCommand is called.
+ * History can also be cleared,
```

Comment:
can we put as much into 1 line? i dont think we need a new line for each sentence

Reply:
This one abit long so i think cannot tho unless i cut

Reply:
I condensed a bit but into 1 single line would violate code quality ("Line too long!")